client/components/social-buttons/facebook.js

```diff
@@ -88,15 +87,15 @@ class FacebookLoginButton extends Component {
 	render() {
 		return (
 			<button className="button" onClick={ this.handleClick }>
-				<span>
-					<SocialLogo className="social-buttons__logo" icon="facebook" size={ 24 } />
```

Comment:
What do you think about updating our [social-logos repo](https://github.com/Automattic/social-logos) to include the color logos? We can do it in a future PR.